idd/Energy+.idd.in

```diff
@@ -24351,6 +24356,7 @@ Exterior:FuelEquipment,
        \required-field
        \type choice
        \key Electricity
+       \key Water
```

Comment:
https://github.com/NREL/EnergyPlus/blob/7fbecf75f88465327b08a4aa03a9a9424c22e889/testfiles/ExteriorLightsAndEq.idf#L229

This is the IDF that caused this issue to come up in our input processing.